packages/safe-apps-sdk/src/types.ts

```diff
@@ -59,11 +59,6 @@ export interface SdkInstance {
 }
 
 export interface SafeInfo {
-  safeAddress: string;
-  network: LowercaseNetworks;
-}
-
-export interface SafeInfoV1 {
   safeAddress: string;
   network: UppercaseNetworks;
```

Comment:
Why is it not possible to have case insensitive networks?

Reply:
Safe interface stores network names in uppercase and we want the type to be a list of constants (e.g "rinkeby", "mainnet", etc)

Reply:
I think to have it truly case insensitive we need to switch to chain id :P